ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -556,6 +557,22 @@ public void removeAnimatorListener(Animator.AnimatorListener animatorListener) {
         animator.removeListener(animatorListener);
     }
 
+    /*
+     * Exposing method to add or remove blur mask filter to ticker text.
+     *
+     * @param style: Blur mask filter type
```

Comment:
don't need the `:` in params doc i dont think

Reply:
My bad, will do the correction.

Also, I created this pull request from a testing account. I'll close this one and create a new one with this correction + resolving the merge conflicts. Sorry about that.